training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+
+More information about Ansible can be found on their website - https://www.ansible.com/overview/how-ansible-works
+
+## Concourse
+Visit https://concourse.onsdigital.co.uk to access Concourse
+
+[<img src="images/concourse-home.png" title="Concourse Homepage" width=49.7% height=49.7%/>][concourse-ons] [<img src="images/concourse-pipeline.png" title="Concourse Pipeline" width=49.7% height=49.7%/>][concourse-ons]
+
+Concourse lists all of the applications where each application has its own [pipeline][concourse-pipeline]. 
+When a [PR][git-pr] of a application is said to be merged into `develop` or `master`, Concourse runs the pipeline of the application which does unit testing, checks if a [docker image][docker-image] can be build for it, bundles it for release and ships it to that release whilst ensuring that it has deployed everything completely.
+
+Furthermore, Concourse helps us to identify if an application has been fully deployed with any new changes implemented and can provide information on the version of the application which is currently running in the `develop` and `production` environments. 
+
+Please note that when deploying changes to the `production` environment (the live website), you need to **manually** deploy the application by doing the following:
+1. Go to [Concourse][concourse-ons]
+2. Navigate to the application
+3. Click on `production-ship-it`
+4. <div>Click on the <img src="images/concourse-build.png" title="Concourse Build" width=4% height=4%></div>
+5. Login to concourse as `dp` user (if you do not have the login, please ask anyone in the team)
+
+More information about Concourse can be found on their website - https://concourse-ci.org/docs.html 
+
+## Consul
+Visit the following to access Consul
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://consul.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://consul.production.onsdigital.co.uk
+
+[<img src="images/consul-dev-services.png" title="Consul Develop Home" width=49.7% height=49.7%/>][consul-dev-services] [<img src="images/consul-dev-nodes.png" title="Consul Develop Nodes" width=49.7% height=49.7%/>][consul-dev-nodes]
+
+> Consul is a service mesh solution providing a full featured control plane with service discovery, configuration, and segmentation functionality. Each of these features can be used individually as needed, or they can be used together to build a full service mesh
+
+Consul is mainly used to check the health of applications which Consul retrieves by calling the health check endpoints of the application (`/health`). This information is then used to monitor cluster health, and it is used by the service discovery components to route traffic away from unhealthy hosts
+
+More information about Consul can be found on their website - https://www.consul.io/docs/intro
+
+## Nomad
```

Comment:
Might it be useful at some point to point out that we sometimes refer to the 'Hashistack', and that is Nomad, Consul and Vault? I dont know where exactly, just thinking it might be a term people hear? (Maybe we should stop using the term so we dont have to explain it 🤷‍♀️  )